### PR TITLE
Silence logs by default

### PR DESCRIPTION
**Pull request by meatballhat**
and allow showing at actions "info" level through `show-logs`

### CODE REVIEW

File: connect-publish/src/connect-publish.ts [hunk 1]
```diff
@@ -237,7 +246,9 @@ export function loadArgs (): ActionArgs {
     dirs.push('.')
   }
 
-  const force = ['true', 'yes', 'ok', 'on'].includes(core.getInput('force').toLowerCase().trim())
+  const truthy = ['true', 'yes', 'ok', 'on']
```
Comment (trestletech) [inline connect-publish/src/connect-publish.ts]: :( APIs are hard.

**Review by trestletech — APPROVED** [review]
🎉

File: connect-publish/src/connect-publish.ts (hunk 1 above)
Comment (meatballhat) [reply connect-publish/src/connect-publish.ts]: It's a bit weird to me that the `@actions/core` doesn't provide something like `core.getInputBoolean` but :shrug: Maybe it's sufficient to use `JSON.parse` if the `YAML bool` => `actions stuff` => `env var` serialization always turns YAML-recognized booleans into `true|false` :thinking:

Comment (meatballhat) [reply connect-publish/src/connect-publish.ts]: Removing `ok` from the list is fine with me :thinking: 

```
$ cat ~/Downloads/manybools.yml
bools:
  yes: yes
  'on': on
  'ok': ok
  'true': true
  'no': no
  'off': off
  'false': false
$ ruby -ryaml -rjson -e 'puts JSON.pretty_generate(YAML.load_file(File.expand_path("~/Downloads/manybools.yml")))'
{
  "bools": {
    "true": true,
    "on": true,
    "ok": "ok",
    "true": true,
    "no": false,
    "off": false,
    "false": false
  }
}
```

**Pull request merged**